word_guess.rb

```diff
@@ -0,0 +1,127 @@
+require "colorize"
+require "rainbow"
+require "artii"
+
+class WordGuess
+  attr_reader :word, :wrong_letters, :dashes
+
+  def initialize
+    @word = get_word
+    @dashes = dashes_array
+    @wrong_letters = [] #empty array to hold used letters
+  end
+
+  def get_word #using this method to use in initialize to call array
+    list_of_words = ["development", "digestion", "window", "yesterday", "separate", "disease", "carriage", "punishment", "transport", "tomorrow", "instrument", "competition", "representative",
+      "achievement", "stomach", "observation", "tendency", "building", "business", "attention", "attraction", "behavior", "questionnaire", "susceptible", "lollipop", "elephant", "hotdog", "awesome",
+      "kilometer", "treasure", "dominoes", "electricity", "lightsaber", "battery", "circumference", "eloquent", "impeccable", "meticulous", "predilection", "superfluous", "vociferous", "zenith"].shuffle #shuffle the array so it's different each time
+    word_array = list_of_words[0].split(//) #always picking first word in array
+  end
+
+  def dashes_array
+    length = "_" * @word.length #.each use regex to check letter, if it's a letter...
+    dash_array = length.split(//)#// splits on a character
+    return dash_array
+  end
+
+  def add_to_wrong_letters(letter)
+    unless @word.include?(letter)
+      @wrong_letters << letter
+    end
+  end
+
+  def display_word(letter = nil)
+    if !@word.include?(letter) && letter != nil && !@wrong_letters.include?(letter)
+      add_to_wrong_letters(letter)
+    end #if letter hasn't been guessed already, and it's not nothing, and it's not in the word
```

Comment:
It is preferred for readability to add an extra line of whitespace after each `end`. So here, after line 41, 45, 50 etc.